### ISSUE TITLE
Device not compatible com.google.ar.core.depth

### ISSUE THREAD

**Issue by angelaumenta** [google-ar/arcore-android-sdk#1563]
### SPECIFIC ISSUE ENCOUNTERED
I've published an APP and lot of devices are not compatible because not have com.google.ar.core.depth. In my APP (made with Unity), the Google ARCore SDK Depth is Optional. For example Samsung galaxy A23 5G with API 12 can not download the APP.

### VERSIONS USED
- Android Studio: 2019 Versión 16.11.8
- ARCore SDK for Android: Version 5.0.3 - November 15, 2022
- Device manufacturer, model, and O/S: Samsung galaxy A23 5G
- Google Play Services for AR (ARCore):  -
  On Windows, use: `adb shell pm dump com.google.ar.core | findstr /i "packages: versionName"`
  On macOS, use: `adb shell pm dump com.google.ar.core | egrep -i versionName\|packages:`
- Output of `adb shell getprop ro.build.fingerprint`: 

### STEPS TO REPRODUCE THE ISSUE
 1. Try to download https://play.google.com/store/apps/details?id=com.Banak.Banak&gl=ES
 2. Use mobile phone without com.google.ar.core.depth funcionality
 

### WORKAROUNDS (IF ANY)
Not posible to download

**Comment by devbridie** [comment]
Hi,

It seems like your application has the following lines in the `AndroidManifest`:

```
    <uses-feature
        android:name="com.google.ar.core.depth"
        android:required="true" />
```

This tells Play Store that your app is not compatible with devices that do not have support for the Depth API. The Samsung Galaxy A23 5G is such a device. You can use the [APK Analyzer](https://developer.android.com/studio/debug/apk-analyzer) to figure out which tags are in your built manifest.

See [Configure your app to be Depth Required or Depth Optional (Android only)](https://developers.google.com/ar/develop/unity-arf/depth/developer-guide#configure_your_app_to_be_depth_required_or_depth_optional_android_only).

**Comment by Tech-AumentaSolutions** [comment]
Hi, I Unity I have activated in Project Settings > Player > Publishing Settings > Custom Main Manifest, so when I've changed in Project Settings > XR Plug-in Management > Google ARCore > Depth to Optional, AndroidManifest was not overwrite. I've unchecked Custom Main Manifest and now using APK Analyzer, Depth did not appears to be true.
You are right and now my APP is compatible with ARCore without Depth API.
Thanks a lot @devbridie ... IS SOLVED !!
